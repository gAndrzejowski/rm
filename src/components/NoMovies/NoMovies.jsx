import React from 'react';
import styles from './NoMovies.scss';

const NoMovies = () => (
    <article className={styles.noResults}>
        <h2>No films found</h2>
    </article>
);
export default NoMovies;
