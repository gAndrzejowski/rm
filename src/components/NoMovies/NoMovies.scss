.noResults {
  font-size:36pt;
  text-align:center;
  width: 100%;
}