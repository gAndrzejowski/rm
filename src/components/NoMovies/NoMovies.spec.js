import React from 'react';
import renderer from 'react-test-renderer';
import NoMovies from './NoMovies';

describe('NoMovies component', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<NoMovies />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
