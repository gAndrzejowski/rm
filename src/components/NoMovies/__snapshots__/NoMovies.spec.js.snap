// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NoMovies component renders correctly 1`] = `
<h2
  className="noResults"
>
  No films found
</h2>
`;
